### PR TITLE
Add redirect for /sign-in on www

### PR DESCRIPTION
**Pull request by joshenlim**

### CODE REVIEW

**Review by alaister — APPROVED** [review]

File: apps/www/lib/redirects.js [hunk 1]
```diff
@@ -1545,4 +1545,9 @@ module.exports = [
     source: '/docs/guides/platform/disk-usage',
     destination: '/docs/guides/platform/database-usage',
   },
+  {
+    permanent: true,
+    source: '/sign-in',
+    destination: 'https://app.supabase.com/sign-in',
+  },
```
Comment (MildTomato) [inline apps/www/lib/redirects.js:1552]: missing 'sign-up' redirect

**Review by MildTomato — CHANGES_REQUESTED** [review]

File: apps/www/lib/redirects.js (hunk 1 above)
Comment (joshenlim) [reply apps/www/lib/redirects.js:1552]: gotcha, added

Comment (alaister) [reply apps/www/lib/redirects.js:1552]: Worth adding forgot password, too, then?

Comment (joshenlim) [reply apps/www/lib/redirects.js:1552]: yup! added

**Review by MildTomato — APPROVED** [review]

**Pull request merged**